cmd/soroban-cli/src/commands/tx/mod.rs

```diff
@@ -17,6 +17,15 @@ pub use args::Args;
 pub enum Cmd {
     /// Edit a transaction envelope from stdin. This command respects the environment variables
     /// `STELLAR_EDITOR`, `EDITOR` and `VISUAL`, in that order.
+    ///
+    /// Example: Start a new edit session
+    ///
+    /// $ stellar tx edit
+    ///
+    /// Example: Pipe an XDR transaction envelope
+    ///
+    /// $ stellar tx new manage-data --data-name hello --build-only | stellar tx edit
```

Comment:
I think we should use a bump seq as an example. It results in no data left on chain. Ultimately payments would be the best example, but it requires too many parameters I think.